lib/routes/snapshots.js

```diff
@@ -38,8 +39,12 @@ router.get('/:snapshotId', function (req, res) {
 });
 
 router.get('/', function (req, res) {
-  req.log.info('Request to get all snapshots');
-  snapshots.getSnapshots({}, function onGetSnapshots (err, snapshots) {
+  var host = { host: req.query.host || undefined };
+  var url = { requestedUrl: req.query.url || undefined };
+  var o = _.merge(host, url);
+  var opts = _.omitBy(o, _.isNil);
+  req.log.info(opts, `Request to search snapshots`);
```

Comment:
I think it would be helpful to log not just the `opts` that will be used for filtering, but also the original query parameters (`req.query`).  This will make our lives better when building out the frontend if we typo something :)